### DISCUSSION TITLE
Recuperación de cuentas

### DISCUSSION THREAD

**Discussion by H4nz3l** [zebnat/morpho-dimension#27] [Ideas]
Se dice que todos merecemos una segunda oportunidad, ¿Se aplica lo mismo para aquellos que fueron banneados?

Expondré un ejemplo: Muchos usuarios al compartir su contraseña con una persona y esta misma comenzara a insultar desde su cuenta, recibieron ban directamente pese a tener todos los corazones enteros

Aún cuando el dueño original cambio la contraseña y nunca había roto las normas (salvo cuando entraron una vez a su cuenta para insultar)

¿No es un poco injusta esa situación?

Con lo cual, se me ocurre la idea de crear un sistema donde puedas pagar con macromonedas para que desbanneen una cuenta

Evidentemente habrán ciertos requisitos para evitar que se abuse de este sistema:

1. Reducir el nivel y/o las monedas del usuario que será desbanneado. Algún castigo deben conservar.

2. El precio a pagar para que un usuario sea desbanneado, será de 5 o 10 millones de macromonedas

(Se podrán hacer donaciones colectivas para poder llegar a esa cantidad de monedas).

3. Solamente las cuentas que hayan estado banneadas por más de 3 años podrán ser desbanneadas mediante un pago.

4. Si el usuario que fue desbanneado incumple las normas nuevamente, se le banneara directamente.

5. Un usuario solo podrá ser desbanneaneado una única vez. Es decir, se trata de una oportunidad excepcional.

- https://www.macrojuegos.com/perfil/h4ns3l/

**Comment by zebnat** [comment]
Estoy a favor de las segundas oportunidades, pero por eso hay un sistema de corazones. El problema es que si la sanción es muy grave el ban es directo.

No se debería hacer sanciones tan graves, pero la razón que me has puesto de ejemplo es que compartieron la cuenta, así que el error es ya de base. No se debería nunca compartir la cuenta con nadie. Incluso si confías en esa persona, a lo mejor hace `Login` desde un ordenador público y se olvida de hacer `logout`.

Igualmente me gusta la propuesta de que sea un rescate colectivo, así que escucharemos.
